crates/common/src/lib.rs

```diff
@@ -464,6 +464,10 @@ impl StarknetVersion {
     }
 
     pub const V_0_13_2: Self = Self::new(0, 13, 2, 0);
+
+    // TODO: version at which block hash definition changes taken from
+    // Starkware implementation but might yet change
```

Comment:
Not sure this comment is relevant here. The constant won't change its value.

Reply:
no, but it might be replaced by a different constant...